advanced/helm/is-with-analytics/mysql-is-with-analytics/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@Savidude please update the license header year.

Reply:
Added https://github.com/wso2/kubernetes-is/pull/145/commits/888cecf9114efe4f696c0277ed3b710684e7e222